homeassistant/components/greeneye_monitor/sensor.py

```diff
@@ -316,7 +315,6 @@ def native_unit_of_measurement(self) -> str:
 class VoltageSensor(GEMSensor[greeneye.monitor.Monitor]):
     """Entity showing voltage."""
 
-    _attr_icon = VOLTAGE_ICON
```

Comment:
You can remove the constants themselves too, since they're no longer used. Also might as well remove the temperature icon while you're at it, since that already has a device class.